### PR TITLE
Stands out

### PR DESCRIPTION
**Pull request by tenisakb**

### CODE REVIEW

File: src/components/stands-out/stands-out-slide.tsx [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+  font-style: ${props => props.theme.fontStyle};
+  font-stretch: ${props => props.theme.fontStretch};
+  line-height: ${props => props.theme.lineHeight};
+  letter-spacing: ${props => props.theme.letterSpacing};
+  color: ${props => props.theme.title.color};
+  text-align: center;
+  margin-top: 100px;
+`;
+
+const H2 = styled.h2`
```
Comment (EdmundMai) [inline src/components/stands-out/stands-out-slide.tsx]: don't name these after HTML tags. name them something descriptive of their purpose

File: src/components/stands-out/stands-out-slide.tsx [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+const RecommendTitle = styled(H2)`
+  font-size: 18px;
+  margin-bottom: 20px;
+`;
+
+const FooterTitle = styled(H2)`
+  margin-top: 80px;
+`;
+
+const P = styled.p`
```
Comment (EdmundMai) [inline src/components/stands-out/stands-out-slide.tsx]: same

File: src/components/stands-out/stands-out-slide.tsx [hunk 3]
```diff
@@ -0,0 +1,496 @@
...
+const Container = styled.div`
+  max-width: 950px;
+  margin: auto;
+  background-color: ${props => props.theme.background};
+  @media only screen and (max-width: ${DESKTOP_WIDTH}px) {
+    padding: 25px 35px;
+  }
+`;
+
+const FlexContainer = styled.div`
```
Comment (EdmundMai) [inline src/components/stands-out/stands-out-slide.tsx]: same

File: src/components/stands-out/stands-out-slide.tsx [hunk 4]
```diff
@@ -0,0 +1,496 @@
...
+  display: flex;
+  justify-content: space-between;
+  margin: 0 0 85px 0;
+  @media (max-width: ${DESKTOP_WIDTH}px) {
+    display: block;
+  }
+`;
+
+const FooterFlexContainer = styled(FlexContainer)`
```
Comment (EdmundMai) [inline src/components/stands-out/stands-out-slide.tsx]: same

File: src/components/stands-out/stands-out-slide.tsx [hunk 5]
```diff
@@ -0,0 +1,496 @@
...
+  justify-content: space-evenly;
+  margin: 100px 0;
+`;
+
+const FlexPasswordManager = styled(FlexContainer)`
```
Comment (EdmundMai) [inline src/components/stands-out/stands-out-slide.tsx]: PasswordManager

File: src/components/stands-out/stands-out-slide.tsx [hunk 6]
```diff
@@ -0,0 +1,496 @@
...
+  margin: 50px 0 150px 0;
+`;
+
+const FooterCollum = styled.div`
```
Comment (EdmundMai) [inline src/components/stands-out/stands-out-slide.tsx]: FooterColumn

File: src/components/stands-out/stands-out-slide.tsx [hunk 7]
```diff
@@ -0,0 +1,496 @@
...
+  display: flex;
+`;
+
+const Collum = styled.div`
```
Comment (EdmundMai) [inline src/components/stands-out/stands-out-slide.tsx]: Column

File: src/components/stands-out/stands-out-slide.tsx [hunk 8]
```diff
@@ -0,0 +1,496 @@
...
+    &:last-child {
+      border-bottom: none;
+    }
+  }
+  @media only screen and (max-width: ${DESKTOP_WIDTH}px) and (min-width: ${MOBILE_WIDTH}px) {
+    height: 230px;
+  }
+`;
+
+const CollumPasswordManager = styled(Collum)`
```
Comment (EdmundMai) [inline src/components/stands-out/stands-out-slide.tsx]: Column

File: src/components/stands-out/stands-out-slide.tsx [hunk 9]
```diff
@@ -0,0 +1,496 @@
...
+          </Collum>
+        </FlexContainer>
+      </Container>
+      <Footer>
+        <FooterTitle>Are you ready to join our thriving community?</FooterTitle>
+        <FooterFlexContainer>
+          <FooterWrapper>
+            <FooterContent>Number of Unique Opacity Accounts</FooterContent>
+            <FooterCollum>
+              <Input />
```
Comment (EdmundMai) [inline src/components/stands-out/stands-out-slide.tsx]: are these functional?

File: src/components/stands-out/stands-out-slide.tsx (hunk 1 above)
Comment (tenisakb) [reply src/components/stands-out/stands-out-slide.tsx]: solved

File: src/components/stands-out/stands-out-slide.tsx (hunk 2 above)
Comment (tenisakb) [reply src/components/stands-out/stands-out-slide.tsx]: solved

File: src/components/stands-out/stands-out-slide.tsx (hunk 3 above)
Comment (tenisakb) [reply src/components/stands-out/stands-out-slide.tsx]: solved

File: src/components/stands-out/stands-out-slide.tsx (hunk 4 above)
Comment (tenisakb) [reply src/components/stands-out/stands-out-slide.tsx]: solved

File: src/components/stands-out/stands-out-slide.tsx (hunk 5 above)
Comment (tenisakb) [reply src/components/stands-out/stands-out-slide.tsx]: solved

File: src/components/stands-out/stands-out-slide.tsx (hunk 6 above)
Comment (tenisakb) [reply src/components/stands-out/stands-out-slide.tsx]: solved

File: src/components/stands-out/stands-out-slide.tsx (hunk 7 above)
Comment (tenisakb) [reply src/components/stands-out/stands-out-slide.tsx]: solved

File: src/components/stands-out/stands-out-slide.tsx (hunk 8 above)
Comment (tenisakb) [reply src/components/stands-out/stands-out-slide.tsx]: solved

File: src/components/stands-out/stands-out-slide.tsx (hunk 9 above)
Comment (tenisakb) [reply src/components/stands-out/stands-out-slide.tsx]: solved

Comment (tenisakb) [reply src/components/stands-out/stands-out-slide.tsx]: I am not sure what are you asking... I dont know what functionality it should have, its without logic.

**Pull request merged**